bundle/manifests/vault-config-operator.clusterserviceversion.yaml

```diff
@@ -505,7 +697,7 @@ spec:
                 - --v=10
                 - --tls-cert-file=/etc/certs/tls/tls.crt
                 - --tls-private-key-file=/etc/certs/tls/tls.key
-                image: registry.redhat.io/openshift4/ose-kube-rbac-proxy@sha256:6d57bfd91fac9b68eb72d27226bc297472ceb136c996628b845ecc54a48b31cb
+                image: quay.io/redhat-cop/kube-rbac-proxy:v0.11.0
```

Comment:
this must have come from a rushed merge, all of this should not be here....

Reply:
Removed.